app/namespace.go

```diff
@@ -5,24 +5,45 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
+	"github.com/temporalio/tcld/protogen/api/auth/v1"
 	"io/ioutil"
+	"net/mail"
 	"strings"
 
 	"github.com/kylelemons/godebug/diff"
+	"github.com/temporalio/tcld/protogen/api/authservice/v1"
 	"github.com/temporalio/tcld/protogen/api/namespace/v1"
 	"github.com/temporalio/tcld/protogen/api/namespaceservice/v1"
 	"github.com/urfave/cli/v2"
 	"google.golang.org/grpc"
 )
 
 const (
-	CaCertificateFlagName     = "ca-certificate"
-	CaCertificateFileFlagName = "ca-certificate-file"
-
+	namespaceRegionFlagName          = "region"
+	namespaceEnvironmentFlagName     = "environment"
+	CaCertificateFlagName            = "ca-certificate"
+	CaCertificateFileFlagName        = "ca-certificate-file"
 	caCertificateFingerprintFlagName = "ca-certificate-fingerprint"
+	certificateFilterFileFlagName    = "certificate-filter-file"
+	certificateFilterInputFlagName   = "certificate-filter-input"
+	searchAttributeFlagName          = "search-attribute"
+	userNamespacePermissionFlagName  = "user-namespace-permission"
 )
 
 var (
+	namespaceRegionFlag = &cli.StringFlag{
+		Name:     namespaceRegionFlagName,
+		Usage:    "Create namespace in this region",
+		Aliases:  []string{"re"},
+		Required: true,
+	}
+	namespaceEnvironmentFlag = &cli.StringFlag{
```

Comment:
what's the point of even adding this if its hidden right now? I believe server will auto-assign the value if its not set anyway, right?

Reply:
removed the flag--you are right it seems to work without it